src/typeDefs/types.ts

```diff
@@ -151,10 +151,34 @@ export const types = gql`
     updatedAt: DateTime!
   }
 
+  type AdvertisementEdge {
+    cursor: String
+    node: Advertisement
+  }
+
+  type AdvertisementsConnection {
+    edges: [AdvertisementEdge]
+    pageInfo: ConnectionPageInfo
+    totalCount: Int
+  }
+
+  extend type Organization {
```

Comment:
You have exposed `Organization.advertisements` two times in the same schema, delete this one.